views/admin/new_event.erb

```diff
@@ -113,6 +113,25 @@
                 </div>
                 <br>
 
+                <div class="dcf-form-group dcf-input-group">
+			        <div class="dcf-input-checkbox">
+			            <input type="checkbox" <%= 'checked="checked"' if event.has_authorization %> id="authorize_tools_checkbox" name="authorize_tools_checkbox">
+			            <label title="Authorize Tools" for="authorize_tools_checkbox"><span class="dcf-sr-only">Authorize Tools</span></label>
+			        </div>
+                    <div class="dcf-form-group">
+                        <label for="specific-group">Authorize Tool(s) (use Shift or Ctrl/Cmd to select multiple):</label>
+                        <select class="dcf-txt-sm" multiple="multiple" id="specific-tools" name="specific_tools[]" >
+                            <% all_tools.each do |tool| %>
+                                <% if create%>
```

Comment:
I think you can remove the ```create``` variable and replace this with ```<% if event.id.nil? %>```. If the event id is null then the page should be creating a new event.